tgui/packages/tgui/interfaces/ChemHeater.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author CodeJester (https://github.com/codeJester27)
+ * @license ISC
+ */
+
+import { classes } from 'common/react';
+import { useBackend } from "../backend";
+import { AnimatedNumber, Box, Button, Dimmer, Icon, Knob, Section, SectionEx, Stack } from '../components';
+import { Window } from '../layouts';
+import { freezeTemperature, getTemperatureColor, getTemperatureIcon, getTemperatureChangeName } from './common/temperatureUtils.js';
+import { NoContainer, ReagentGraph, ReagentList } from './common/ReagentInfo.js';
```

Comment:
```suggestion
import { NoContainer, ReagentGraph, ReagentList } from './common/ReagentInfo';
```
If `.js` *is* needed for this to work, please let me know and I'll take a look into why (and not block this PR on it).